src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -313,6 +323,11 @@ protected static void MergeExistingInMemoryProfiles(LaunchSettingsData newSnapsh
                     {
                         // Create a new one from the existing in-memory profile and insert it in the same location, or the end if it
                         // is beyond the end of the list
+                        if (newSnapshot.Profiles == null)
```

Comment:
>newSnapshot.Profiles == null [](start = 28, length = 28)

Is this ever true or would `FirstOrDefault` have thrown `ArgumentNullException` in that case? Consider replacing with `Debug.Assert(newSnapshot.Profiles != null);` which should prevent subsequent warnings.

Reply:
Good catch - thanks.